File: bloco_04/dia_4/lista-de-exercicios/funcoes/ex4.js
function maiorQuantidadeCaracteres (nomes){
  let maisCaracteres = 0;
  let index = 0;
  for(let i in nomes){
    if(nomes[i].length > maisCaracteres){
      maisCaracteres = nomes[i].length;
      index = i;
    }    
  }
  return console.log(nomes[index]);
}

maiorQuantidadeCaracteres(['José', 'Lucas', 'Nádia', 'Fernanda', 'Cairo', 'Joana']);

File: bloco_33/dia_1/Lista_de_exercicios/ex4.py
def returnBiggerName(list):
  biggerName=list[0]
  for item in list:
    if len(item) > len(biggerName):
      biggerName = item
  return biggerName

print(returnBiggerName(["José", "Lucas", "Nádia", "Fernanda", "Cairo", "Joana"]))